docs: tidying up the Cell geolocations doc (#142)
NordicSemiconductor/asset-tracker-cloud-docs: docs/guides/CellGeolocations.rst

This allows to offload expensive calculations to the cloud and reduce the resource usage on the device.

In the nRF Asset Tracker, whenever a device reports a GPS fix, the position is stored together with the cell ID.
An up-to-date cell location database independent of a third-party API is built over time.
This database is used as the primary means of geolocating cells in the nRF Asset Tracker.
The third-party API will be called (if it is enabled) only if a cell has not been geolocated by a device previously.

Geolocating cells using third-party APIs
========================================

> Third-party APIs and services like `UnwiredLabs <https://unwiredlabs.com/>`_ and `CellMapper <https://www.cellmapper.net/>`_ have a database of cell tower locations and provide an API to query from these locations.

Reviewer: This changes the meaning. One does not query *from* the physical location of the cell tower, but the cell tower location is looked up in their database.

> Third-party APIs and services like `UnwiredLabs <https://unwiredlabs.com/>`_ and `CellMapper <https://www.cellmapper.net/>`_ have a database of cell tower locations and provide an API to query from these locations.

Reviewer: ```suggestion
Third-party APIs and services like `UnwiredLabs <https://unwiredlabs.com/>`_ and `CellMapper <https://www.cellmapper.net/>`_ have a database of cell tower locations and provide an API to query the cell tower locations.
```
